draft-packages/modal/docs/Modal.stories.tsx

```diff
@@ -32,6 +33,16 @@ import { figmaEmbed } from "../../../storybook/helpers"
 
 import styles from "./Modal.stories.scss"
 
+// Add additional height for Chromatic snapshots only
```

Comment:
I wouldn't mind a comment explaining why a 100vh container is important for chromatic. Is it easy to explain?